compair/static/less/print.less

```diff
@@ -102,20 +140,13 @@
     .rich-content-embed {
         display: none;
     }
-    .each-attachment-content .heading a {
-        display: transparent;
-        border: none;
-    }
-    .each-attachment-content .heading a:after {
-        content: ' [Log in to the application to see and download this file]';
+    .rich-content a {
+        text-decoration: underline;
     }
-
-    /* add space for writing next to answers */
-    .assignment-screen .tab-content .each-answer > .content,
-    .assignment-screen .tab-content .each-comment > .content,
-    .evaluations-screen .each-evaluation .content {
-        max-height: none;
-        width: 80%;
+    .rich-content a[href]:after {
+        content: ' [see: ' attr(href) '] ';
+        font-size: 80%;
+        overflow-wrap: break-word;
     }
 
```

Comment:
Please add
```
a[href=""]:after {
    content: "";
}
```

to get rid of the `()` that bootstrap adds when `href=""` (happens for some of our expandable content links)